pyproject.toml

```diff
@@ -52,6 +52,10 @@ include_trailing_comma = true
 combine_as_imports = true
 skip_gitignore = true
 
+[tool.maturin]
+python-source = "../synapse"
```

Comment:
I assume this is referring to the name of the checkout of the repo rather than the repo's `synapse` directory? If so can you add a comment about this because I would find it a bit confusing.

Reply:
Oooh, turns out that isn't needed anymore. :tada: